docs/reference.md

```diff
@@ -845,6 +845,22 @@ message returned.
 
 Example: _2021-08-31T15:50:00Z_
 
+### get-attachment
+
+Downloads an attachment to file system and returns its path.
```

Comment:
we should highlight that we do not clean the file and that it is up to the workflow. writer to do it
Or should we delete them when the JVM exists?

Reply:
done